.circleci/config.yml

```diff
@@ -62,7 +62,7 @@ version: 2.1
   run:
     name: Upload coverage results to Code Climate
     command: |
-      coverage/test-reporter sum-coverage coverage/codeclimate.*.json --parts 19 --output coverage/codeclimate.total.json
+      coverage/test-reporter sum-coverage coverage/codeclimate.*.json --parts 22 --output coverage/codeclimate.total.json
```

Comment:
@ionut998 would you mind adding a comment above this? @deivid-rodriguez I saw your comment about this change and didn't remember what it was, thanks. Is the parts number the total count of builds/jobs plus 1? Is the plus 1 the lint job? Maybe a note like:

```
# The parts number is the total jobs count plus 1 (for the lint job)
coverage/test-reporter sum-coverage coverage/codeclimate.*.json --parts 22 --output coverage/codeclimate.total.json
```

Reply:
Yeah, that's it. It's the number of jobs that generate coverage reports.

Reply:
Added the comment